namecheap.py

```diff
@@ -15,6 +16,10 @@
 }
 NAMESPACE = "http://api.namecheap.com/xml.response"
 
+# default values for the retry mechanism
```

Comment:
I would like to rework those "retries" as "attempts" to replace following `while True` conditional with `while attempts > 0`, which looks more readable for me.